instrumentation/logback/logback-appender-1.0/library/src/slf4j2ApiTest/resources/logback-test.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+  <appender name="console" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>
+        %d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
+      </pattern>
+    </encoder>
+  </appender>
+  <appender name="OpenTelemetry"
+    class="io.opentelemetry.instrumentation.logback.appender.v1_0.OpenTelemetryAppender">
+    <captureExperimentalAttributes>false</captureExperimentalAttributes>
+    <captureCodeAttributes>true</captureCodeAttributes>
+    <captureMarkerAttribute>true</captureMarkerAttribute>
+    <captureMdcAttributes>*</captureMdcAttributes>
```

Comment:
How come this doesn't need `<captureKeyValuePairAttributes>true</captureKeyValuePairAttributes>`?

Reply:
Thanks, fixed.

Reply:
Eventually, I hope we can make some / all of these enabled by default.